src/utils/authentication/utils/getUrlParameter.js

```diff
@@ -0,0 +1,8 @@
+export default (key) => {
+  const cleanKey = key.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]')
+  const regex = new RegExp(`[\\?&]${cleanKey}=([^&#]*)`)
```

Comment:
I agree, well named method and file gives me all the hints I need to understand what's what in this circumstance.